docs/en/05_Contributing/04_Making_A_SilverStripe_Core_Release.md

```diff
@@ -138,6 +144,13 @@ main sets of commands:
 
 ### Stage 1: Release preparation:
 
+If you are managing a release, it's best to first make sure that SilverStripe marketing
+are aware of any impending release. This is so that they can ensure that a relevant blog
+post will appear on [www.silverstripe.org/blog](http://www.silverstripe.org/blog/), and
+cross-posted to other relevant channels such as Twitter and Facebook.
+Make sure Nicole and Vinh at silverstripe at aware by sending an email to
```

Comment:
I wouldn't mention specific people here, it just creates a maintenance hassle on the long run :) Otherwise thanks for writing those, that's awesome!